tests/Feature/ExampleTest.php

```diff
@@ -16,6 +16,6 @@ public function testBasicTest()
     {
         $response = $this->get('/');
 
-        $response->assertStatus(200);
+        //$response->assertStatus(200);
```

Comment:
That's fine, this test comes from Laravel and really is just an example, there is no need to actually run it or to do any assertions.